### PR TITLE
[risk=no] [RW-6563] replacing spinner component with react one

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Converting spinner component to react

### CODE REVIEW

**Review by dolbeew — COMMENTED** [review]
We should put simple components like this in utility file, similar to [spinners.tsx](https://github.com/all-of-us/workbench/blob/master/ui/src/app/components/spinners.tsx) in workbench. Obviously we won't be able to use them yet, but we can go ahead and create them, then switch when the parent components are converted.

We'd also need to use inline or object-based styles instead of CSS files, which we probably need to do anyway. Currently, we're still relying on Angular to load the CSS files for wrapped React components.

File: public-ui/src/app/utils/spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    from{background:transparent}
+    to{background:#302c70}
+  }
+`;
+
+
+interface SpinnerProps {
+  loading: boolean;
+  dots: boolean;
+}
```
Comment (dolbeew) [inline public-ui/src/app/utils/spinner.tsx]: I think we can remove both props. `loading` should be handled in the parent and only render the spinner if true. Also, we should split the two types into separate consts. 

So once we convert the parent components, should look something like this:
```
{loading && <Spinner/>}
{loading && <LoadingDots/>} // or whatever we choose to name this one
```

File: public-ui/src/app/utils/spinner.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+export const spinnerCss = `
```
Comment (dolbeew) [inline public-ui/src/app/utils/spinner.tsx:12]: This is fine for now, but may change when we start aligning styles more with workbench. May be good to separate styles used for the different spinners if we split them into separate consts

File: public-ui/src/app/utils/spinner.tsx (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/utils/spinner.tsx]: I split these both into different consts.

File: public-ui/src/app/utils/spinner.tsx (hunk 2 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/utils/spinner.tsx:12]: I have seperated styles. Ll take a look at workbench styles.

File: public-ui/src/app/utils/spinner.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            <div className='loading-dots'>
+            <div className='dot'></div>
+                <div className='dot'></div>
+                <div className='dot'></div>
+                <div className='dot'></div>
+           </div>
+           </React.Fragment>;
+};
+
+export const SpinnerContainer = ({loading, dots}) => {
```
Comment (dolbeew) [inline public-ui/src/app/utils/spinner.tsx]: So, I think we can remove this also. The components above look good and we can just import them individually whenever we need them in other components.

File: public-ui/src/app/utils/spinner.tsx (hunk 2 above)
Comment (dolbeew) [reply public-ui/src/app/utils/spinner.tsx:12]: Yeah, I'm going to try to work on styles some next week. Converting the dots CSS here to objects looks pretty straightforward, not sure about some of the spinner styles like `@keyframes` and `a:link,a:visited`.

**Comment by SrushtiGangireddy** [comment]
yeah for keyframes i see there are some options like using styles-components or css-modules. We can try them.

File: public-ui/src/app/utils/spinner.tsx (hunk 3 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/utils/spinner.tsx]: Got it. thanks.

**Review by dolbeew — APPROVED** [review]

**Pull request merged**